Main/ClientApp/src/Components/WelcomeScreen.tsx

```diff
@@ -1,11 +1,31 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PanelWideMessage from "./Common/PanelWideMessage";
 import Layout from "./Layout";
+import authService from "../Authorization/AuthorizeService";
+import { Redirect } from "react-router-dom";
 
 export default function WelcomeScreen() {
```

Comment:
BTW, I had an idea to display some screenshots/tutorials on the home page with a button "Get started" that would either redirect to collections or login page. 
This is also fine, but if I have time, I will do a nicer home page

Reply:
that home page would be great, but for now this is better than "Log in to get started" caption